plugins/figura_format/figura_format.js

```diff
@@ -89,14 +56,14 @@
 		icon: "icon.svg",
 		description: "Create models for the Figura mod in a custom format that optimizes Blockbench to work with Figura models.",
 		tags: ["Minecraft: Java Edition", "Figura"],
-		version: "0.1.2",
+		version: "0.1.3",
 		min_version: "4.8.0",
 		variant: "both",
 		await_loading: true,
 		creation_date: "2023-07-22",
 		onload() {
-			let particle = EffectAnimator.prototype.channels.particle, sound = EffectAnimator.prototype.channels.sound
-			let format = new ModelFormat('figura', {
+			const particle = EffectAnimator.prototype.channels.particle, sound = EffectAnimator.prototype.channels.sound
+			const format = new ModelFormat('figura', {
 				icon: 'change_history',
 				name: 'Figura Model',
 				description: 'Model for the Figura mod.',
```

Comment:
It would be good to change the category here to "minecraft", currently it is labeled as "low_poly".